scanner/src/test/java/ca/on/oicr/gsi/runscanner/scanner/processor/RevioPacBioProcessorTest.java

```diff
@@ -0,0 +1,47 @@
+package ca.on.oicr.gsi.runscanner.scanner.processor;
+
+import ca.on.oicr.gsi.runscanner.dto.NotificationDto;
+import ca.on.oicr.gsi.runscanner.dto.PacBioNotificationDto;
+import ca.on.oicr.gsi.runscanner.dto.type.Platform;
+import ca.on.oicr.gsi.runscanner.scanner.processor.RunProcessor.Builder;
+import java.io.File;
+import java.io.IOException;
+import java.util.TimeZone;
+import org.junit.Assert;
+
+public class RevioPacBioProcessorTest<T> extends AbstractProcessorTest<PacBioNotificationDto> {
+  private final RevioPacBioProcessor instance =
+      new RevioPacBioProcessor(new Builder(Platform.PACBIO, "unittest", null));
+  private NotificationDto afterProcessing;
+
+  public RevioPacBioProcessorTest() {
+    super(PacBioNotificationDto.class);
+  }
+
+  @Override
+  protected NotificationDto process(File directory) throws IOException {
+    afterProcessing = instance.process(directory, TimeZone.getTimeZone("America/Toronto"));
+    return afterProcessing;
+    // return instance.process(directory, TimeZone.getTimeZone("America/Toronto"));
+  }
+
+  @Override
+  public void beforeComparison(PacBioNotificationDto reference, PacBioNotificationDto result) {
+    // Check if reference run name matches new run directory name
+    if (afterProcessing.getRunAlias().equals(reference.getRunAlias())) {
```

Comment:
1. This should only be for one specific run, so you should have the name of that run hard-coded here. As is, you're changing the start time of every Revio test run
2. There's no need for`afterProcessing` - it's the same object as `result`